terraform/environment/provider.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    openstack = {
+      source = "terraform-provider-openstack/openstack"
+      version = "1.46.0"
+    }
+  }
+}
+
+provider "openstack" {
+  cloud = "openstack"
```

Comment:
no variable replacement for cloud? how do you handle several projects from same `clouds.yaml` file?

Reply:
I saw it was correctly set from the modules `cloud = "_OS_CLOUD_"`  but not here